lib/utils/archyTree.ts

```diff
@@ -1,4 +1,4 @@
-import * as archy from 'archy';
+import archy = require('archy');
```

Comment:
why this change?

Reply:
While playing around with some ts settings, this change had to be made, but it looks like our current config allows us to keep the original import without require.  Reverting (next push).